x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicator_field/indicator_field.tsx

```diff
@@ -20,19 +21,21 @@ export interface IndicatorFieldProps {
    * The field to get the indicator's value for.
    */
   field: string;
-  /**
-   * An object to know what type the field is ('file', 'date', ...).
-   */
-  fieldTypesMap: { [id: string]: string };
 }
 
 /**
  * Takes an indicator object, a field and a field => type object to returns the correct value to display.
  * @returns If the type is a 'date', returns the {@link DateFormatter} component, else returns the value or {@link EMPTY_VALUE}.
  */
-export const IndicatorField: VFC<IndicatorFieldProps> = ({ indicator, field, fieldTypesMap }) => {
+export const IndicatorField: VFC<IndicatorFieldProps> = ({ indicator, field }) => {
+  const indicatorsTableContext = useContext(IndicatorsTableContext);
+
+  if (!indicatorsTableContext) {
```

Comment:
I'm a bit on the fence about making this quite a generic component to the table context. I know that right now it all kind of make sense as we have all the data coming through the data grid, but in the future we will likely want to open our fly out from other contexts, eg. from Timeline. Maybe that's a problem for the future us though.